include/swift/AST/Attr.def

```diff
@@ -376,12 +392,14 @@ SIMPLE_DECL_ATTR(_show_in_interface, ShowInInterface,
 DECL_ATTR(_cdecl, CDecl,
   OnFunc | OnAccessor |
   LongAttribute | UserInaccessible |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
+  // FIXME: We could make this stable to add by having a separate thunk for the
+  // C entry point, like we do with ObjC entry points.
```

Comment:
We actually do, the same SILGen logic is used to emit the thunk for both. Maybe add an evolution test :)

Reply:
I guess it's also only stable to add if we're not caring about backwards-deployment, but that's consistent with `@objc`. :-/